internal/store/engine_state.go

```diff
@@ -75,6 +75,8 @@ type EngineState struct {
 	LastTiltfileBuild    model.BuildRecord
 	CurrentTiltfileBuild model.BuildRecord
 	TiltfileCombinedLog  model.Log
+
+	SailUrl string
```

Comment:
i thought that this should be SailURL in go style?
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
ah yep, you're right